data-science/us-east-1/genai-llm-rag-bedrock-poc/locals.tf

```diff
@@ -23,17 +23,17 @@ locals {
       environment = [
         {
           "name" : "AWS_DEFAULT_REGION"
-          "value" : "us-east-1"
+          "value" : "us-west-2"
```

Comment:
@martingaleano Tincho shouldn't us-east-1 be the default region. Is this related with the bedrock available models in this region?

Reply:
Exactly @exequielrafaela! The infrastructure is created in `us-east-1` but the app needs to interact with some models that are only available (at least for now) in `us-west-2`. 

I'll leave a comment with this observation.